quadriga/src/main/java/edu/asu/diging/quadriga/core/service/impl/MappedTripleGroupServiceImpl.java

```diff
@@ -68,7 +74,9 @@ public MappedTripleGroup add(String collectionId, MappedTripleType mappedTripleT
     @Override
     public MappedTripleGroup findByCollectionIdAndMappingType(String collectionId, MappedTripleType mappedTripleType)
             throws InvalidObjectIdException, CollectionNotFoundException {
-        Collection collection = collectionManager.findCollection(collectionId);
+        
+        Collection collection = new Collection();
+        collection = collectionManager.findCollection(collectionId);
```

Comment:
line 78 is pretty much useless, since you are overriding the value of `collection` in line 79